<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewCallGraphWidget</class>
 <widget class="QDialog" name="NewCallGraphWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>334</width>
    <height>414</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>QSAGUI - new call graph</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Call graph:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QLineEdit" name="outputFileEdit"/>
     </item>
     <item>
      <widget class="QPushButton" name="outputFileButton">
       <property name="text">
        <string>Browse</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="generateButton">
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton">
       <property name="text">
        <string>Cancel</string>
       </property>
       <property name="shortcut">
        <string>Esc</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="cTab">
      <attribute name="title">
       <string>C</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_5">
       <item row="0" column="0">
        <widget class="QCheckBox" name="baseDirCBox">
         <property name="text">
          <string>Base directory:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,0">
         <property name="sizeConstraint">
          <enum>QLayout::SetDefaultConstraint</enum>
         </property>
         <item>
          <widget class="QLineEdit" name="baseDirEdit">
           <property name="enabled">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="baseDirButton">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Browse</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="2" column="0">
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>Files for parsing</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="0" column="0">
           <widget class="QRadioButton" name="preprocessedFileRButton">
            <property name="text">
             <string>Preprocessed code:</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QListWidget" name="listOfCFiles"/>
          </item>
          <item row="2" column="0">
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="addCButton">
              <property name="text">
               <string>Add...</string>
              </property>
              <property name="icon">
               <iconset resource="../resources/resources.qrc">
                <normaloff>:/images/list-add.png</normaloff>:/images/list-add.png</iconset>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="delCButton">
              <property name="enabled">
               <bool>true</bool>
              </property>
              <property name="text">
               <string>Delete</string>
              </property>
              <property name="icon">
               <iconset resource="../resources/resources.qrc">
                <normaloff>:/images/list-remove.png</normaloff>:/images/list-remove.png</iconset>
              </property>
              <property name="shortcut">
               <string>Del</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="3" column="0">
           <widget class="QRadioButton" name="sourceFileRButton">
            <property name="text">
             <string>Source file (input command for preprocessing):</string>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <widget class="QLineEdit" name="sourceFileEdit">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>gcc &lt;input_file_name&gt; -E -o &lt;output_file_name&gt;</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="assemblerTab">
      <attribute name="title">
       <string>Assembler</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="0" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Source files for parsing:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QListWidget" name="listOfAsmFiles"/>
       </item>
       <item row="2" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_7">
         <item>
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="addAsmButton">
           <property name="text">
            <string>Add...</string>
           </property>
           <property name="icon">
            <iconset resource="../resources/resources.qrc">
             <normaloff>:/images/list-add.png</normaloff>:/images/list-add.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="delAsmButton">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="text">
            <string>Delete</string>
           </property>
           <property name="icon">
            <iconset resource="../resources/resources.qrc">
             <normaloff>:/images/list-remove.png</normaloff>:/images/list-remove.png</iconset>
           </property>
           <property name="shortcut">
            <string>Del</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="fortran90Tab">
      <attribute name="title">
       <string>Fortran90</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Source files for parsing:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QListWidget" name="listOfF90Files"/>
       </item>
       <item row="2" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_8">
         <item>
          <spacer name="horizontalSpacer_4">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="addF90Button">
           <property name="text">
            <string>Add...</string>
           </property>
           <property name="icon">
            <iconset resource="../resources/resources.qrc">
             <normaloff>:/images/list-add.png</normaloff>:/images/list-add.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="delF90Button">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="text">
            <string>Delete</string>
           </property>
           <property name="icon">
            <iconset resource="../resources/resources.qrc">
             <normaloff>:/images/list-remove.png</normaloff>:/images/list-remove.png</iconset>
           </property>
           <property name="shortcut">
            <string>Del</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>tabWidget</tabstop>
  <tabstop>baseDirCBox</tabstop>
  <tabstop>baseDirEdit</tabstop>
  <tabstop>baseDirButton</tabstop>
  <tabstop>preprocessedFileRButton</tabstop>
  <tabstop>listOfCFiles</tabstop>
  <tabstop>addCButton</tabstop>
  <tabstop>delCButton</tabstop>
  <tabstop>sourceFileRButton</tabstop>
  <tabstop>sourceFileEdit</tabstop>
  <tabstop>outputFileEdit</tabstop>
  <tabstop>outputFileButton</tabstop>
  <tabstop>generateButton</tabstop>
  <tabstop>cancelButton</tabstop>
  <tabstop>listOfAsmFiles</tabstop>
  <tabstop>addAsmButton</tabstop>
  <tabstop>delAsmButton</tabstop>
  <tabstop>listOfF90Files</tabstop>
  <tabstop>addF90Button</tabstop>
  <tabstop>delF90Button</tabstop>
 </tabstops>
 <resources>
  <include location="../resources/resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>NewCallGraphWidget</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>314</x>
     <y>391</y>
    </hint>
    <hint type="destinationlabel">
     <x>181</x>
     <y>206</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>baseDirCBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>baseDirEdit</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>180</x>
     <y>49</y>
    </hint>
    <hint type="destinationlabel">
     <x>138</x>
     <y>77</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>baseDirCBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>baseDirButton</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>180</x>
     <y>49</y>
    </hint>
    <hint type="destinationlabel">
     <x>301</x>
     <y>78</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>preprocessedFileRButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>sourceFileEdit</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>180</x>
     <y>126</y>
    </hint>
    <hint type="destinationlabel">
     <x>180</x>
     <y>294</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sourceFileRButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>listOfCFiles</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>180</x>
     <y>268</y>
    </hint>
    <hint type="destinationlabel">
     <x>180</x>
     <y>180</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sourceFileRButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>addCButton</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>180</x>
     <y>268</y>
    </hint>
    <hint type="destinationlabel">
     <x>218</x>
     <y>239</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sourceFileRButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>delCButton</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>180</x>
     <y>268</y>
    </hint>
    <hint type="destinationlabel">
     <x>301</x>
     <y>239</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
